src/common/src/metrics/relabeled_metric.rs

```diff
@@ -26,58 +27,99 @@ use risingwave_common::config::StorageMetricLevel;
 /// We could have use one single struct to represent all `MetricVec<T: MetricVecBuilder>`, rather
 /// than specializing them one by one. However, that's undoable because prometheus crate doesn't
 /// export `MetricVecBuilder` implementation like `HistogramVecBuilder`.
-
 #[derive(Clone, Debug)]
 pub struct RelabeledHistogramVec {
-    relabel_threshold: StorageMetricLevel,
-    metric_level: StorageMetricLevel,
+    relabel_threshold: MetricLevel,
+    metric_level: MetricLevel,
     metric: HistogramVec,
+    relabel_num: usize,
```

Comment:
Could you add some comments about why we need this?

Reply:
Sure.